nautobot/extras/querysets.py

```diff
@@ -83,6 +83,9 @@ def annotate_config_context_data(self):
 
         Order By clause in Subquery is not guaranteed to be respected within the aggregated JSON array, which is why
         we include "weight" and "name" into the result so that we can sort it within Python to ensure correctness.
+
+        TODO This method is broken because of the reasons stated in _get_config_context_filters()
```

Comment:
```suggestion
        TODO This method does not accurately reflect location inheritance because of the reasons stated in _get_config_context_filters()
```